### PR TITLE
feat: Blockchain.deserialize

### PR DESCRIPTION
**Pull request by shruggr**

### CODE REVIEW

File: tsconfig-scryptTS.json [hunk 1]
```diff
@@ -5,6 +5,13 @@
     "noEmit": true,
     "experimentalDecorators": true,
     "target": "ESNext",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "plugins": [
```
Comment (xhliu) [inline tsconfig-scryptTS.json:9]: Is this intended? Looks like a local config.

Comment (shruggr) [reply tsconfig-scryptTS.json:9]: Hmmm. No, and I didn't see that in the changelog when I submitted that.

Comment (xhliu) [reply tsconfig-scryptTS.json:9]: Will merge once this part is removed.

**Review by gitzhou — APPROVED** [review]

**Review by xhliu — APPROVED** [review]
👌

**Pull request merged**